src/assets/icons/brands/smashingmagazine.svg

```diff
@@ -1,4 +1,5 @@
-  <path d="M13.5,7.4 L37.1,2 C36.4,1.8 35.7,1.7 35,1.6 L14.6,6.3 C14.2,6.6 13.9,7 13.5,7.4 L13.5,7.4 Z M45.1,31 C44.4,30.2 43.6,29.4 42.7,28.6 C39.4,30.9 35.1,31.8 29.9,29.3 C25.8,27.3 21.7,27.3 18.1,28.3 C19.9,29.6 22.1,30.7 24.6,31.8 C28.8,33.6 31.7,35.1 33.1,36.4 C34.6,37.7 35.3,39.3 35.3,41.4 C35.3,44.8 32.9,46.6 28,46.6 C23.3,46.6 19.1,45.1 15.4,42.2 L11,52.1 C14.4,53.9 17.1,55.1 19.2,55.6 C19.3,55.6 19.4,55.7 19.5,55.7 L45.2,49.8 C47.2,47.3 48.2,44.2 48.2,40.6 C48.2,36.7 47.2,33.6 45.1,31 L45.1,31 Z M10.8,15.7 C10.8,20.7 13.2,24.9 18.1,28.2 C21.8,27.2 25.8,27.3 29.9,29.2 C35.1,31.7 39.4,30.7 42.7,28.5 C40.7,26.9 38.2,25.5 35,24.2 C30.3,22.3 27.3,20.9 25.9,19.8 C24.5,18.7 23.8,17.5 23.8,16.1 C23.8,12.7 26,10.9 30.3,10.9 C35.1,10.9 38.9,12.1 41.6,14.4 L45.2,4.7 C42.6,3.4 39.9,2.4 37.2,1.8 L13.6,7.2 C11.7,9.7 10.8,12.4 10.8,15.7 Z" fill="#FFFFFF" />
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24px" height="24px">
+  <path d="M22.48,16.85,19.28,3.1a2.07,2.07,0,0,0-2.49-1.56l-2.3.52a5.36,5.36,0,0,1,.78.15l1.3-.3a2.08,2.08,0,0,1,2.49,1.56l1.19,5.09a9.27,9.27,0,0,1-2.86,3.57,7.48,7.48,0,0,1,.89.89,5.47,5.47,0,0,1,1.15,3.53A5.34,5.34,0,0,1,18.32,20L21,19.38a2.16,2.16,0,0,0,1.52-2.53ZM8.11,12l.1,0a11.16,11.16,0,0,0,2.41,1.3A13,13,0,0,1,13.78,15a2.36,2.36,0,0,1,.82,1.86c0,1.3-.93,1.94-2.71,1.94a7.37,7.37,0,0,1-4.68-1.64L5.57,20.79h0a11.61,11.61,0,0,0,3.09,1.26s.07,0,.11,0l-1.56.37A2.07,2.07,0,0,1,4.72,20.9L1.52,7.15A2.09,2.09,0,0,1,3.08,4.66l3.83-.89a3.79,3.79,0,0,0-.37.41l-3,.7a2.36,2.36,0,0,0-.74.33A3,3,0,0,1,3.49,5l3-.71a4.74,4.74,0,0,0-1,3.09h0A5.63,5.63,0,0,0,8.11,12Zm8.83-5.08a6.32,6.32,0,0,0-4.2-1.3c-1.59,0-2.41.63-2.41,1.93a1.74,1.74,0,0,0,.78,1.38,15.73,15.73,0,0,0,3.38,1.63,9.91,9.91,0,0,1,2.86,1.6,9.27,9.27,0,0,0,2.86-3.57L19,3.47a2,2,0,0,0-2.49-1.56l-1.3.3a11.55,11.55,0,0,1,3,1.08Z"/>
+  <path d="M6.49,4.19l8.77-2A7.14,7.14,0,0,0,14.48,2L6.9,3.78a4,4,0,0,0-.41.41ZM18.23,13a8.3,8.3,0,0,0-.89-.89,4.5,4.5,0,0,1-4.76.26A6.05,6.05,0,0,0,8.2,12a12.44,12.44,0,0,0,2.42,1.3A12.5,12.5,0,0,1,13.77,15a2.31,2.31,0,0,1,.82,1.86c0,1.27-.89,1.93-2.71,1.93A7.37,7.37,0,0,1,7.2,17.12L5.56,20.8a14.22,14.22,0,0,0,3.05,1.3s.07,0,.11,0l9.55-2.2a5.35,5.35,0,0,0,1.12-3.42A5.41,5.41,0,0,0,18.23,13ZM5.49,7.27A5.45,5.45,0,0,0,8.2,11.92a6.19,6.19,0,0,1,4.38.37A4.54,4.54,0,0,0,17.34,12a11,11,0,0,0-2.86-1.6A16.6,16.6,0,0,1,11.1,8.79a1.71,1.71,0,0,1-.78-1.37c0-1.26.82-1.93,2.41-1.93a6.26,6.26,0,0,1,4.2,1.3l1.34-3.61a13.08,13.08,0,0,0-3-1.07l-8.77,2A5,5,0,0,0,5.49,7.27Z"
+  fill="#fff"/>
```

Comment:
Is the white fill intentional here? Looking at the logo, I think it might be better if we removed the "S" from the path so it was monochrome like the others are?

(Apologies if this was inherited from a previous version, but we should fix it now.)

Also, line 4 is missing indentation.

Reply:
This was inherited from the previous version of the file, but I agree. Let's fix it!

Reply:
Yesh! Upon closer inspection, this asset looks a big rugged! I might quickly browse for a more recent and hopefully cleaner version to work with. 

![Screen Shot 2020-05-15 at 1 01 16 PM](https://user-images.githubusercontent.com/1427548/82091440-37077b80-96ac-11ea-9efc-926b47d00dae.png)

Reply:
This resulted in a much cleaner, icon and a smaller SVG!

![Screen Shot 2020-05-15 at 1 31 50 PM](https://user-images.githubusercontent.com/1427548/82093761-9b2c3e80-96b0-11ea-9f05-c4ad6f8e9940.png)